packages/mgt-components/src/components/mgt-get/mgt-get.ts

```diff
@@ -227,6 +228,8 @@ export class MgtGet extends MgtTemplatedComponent {
    * @memberof MgtGet
    */
   public refresh(hardRefresh = false) {
+    this.refreshType = hardRefresh.toString();
```

Comment:
the `refreshType` property feels awkward, especially since it's a string. I think you can do this without introducing a new propoerty and by using the `isRefreshing` property that is already being set here. However, I think you will need to move setting the `isRefreshing = false` line to the end of the `loadState` method as I don't think it does anything here anyway (`requestStateUpdate` finishes way before `loadState`)

Reply:
clearstate is now used instead:
40b3c35

@nmetulev 

Sorry, I couldn't come up with other `mgt-get` stories (other than 1 new one for testing refresh) that weren't already used or just an extension of what we have.. 
Feel free to pass for now based on the testing. Otherwise I will revisit this soon , just trying to start on other projects.